### PR TITLE
fix(curriculum): update current year

### PR DESCRIPTION
**Pull request by Supravisor**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

Update the text so the example year given is current

### CODE REVIEW

**Review by jdwilkin4 — COMMENTED** [review]
I feel like this might have been discussed earlier and I don't remember where we landed with this. I thought the consensus was to leave it as is instead of updating it each year.

But @nieldakarla might know more about this and if we want to update this each year

**Comment by gikf** [comment]
Related https://github.com/freeCodeCamp/freeCodeCamp/issues/53594

**Comment by ilenia-magoni** [comment]
To avoid people wanting to update it each year, we could put a year that is 20 years in the past or 20 years in the future

**Comment by jdwilkin4** [comment]
Yeah, I like the idea of creating an example based on 20 years into the future

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657ccdeb0919a803c00c8cbd.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ Finally, this is Linda, our UX/UI designer. She joined the team last year.
 
 # --description--
 
-`Last year` refers to the year before the current one. For example, if it is 2023 now, last year was 2022. `This year` refers to the current year, 2023 in this example. 
+`Last year` refers to the year before the current one. For example, if it is 2024 now, last year was 2023. `This year` refers to the current year, 2024 in this example.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657ccdeb0919a803c00c8cbd.md:14]:
```suggestion
`Last year` refers to the year before the current one. For example, if it is 2034 now, last year was 2033. `This year` refers to the current year, 2034 in this example. 
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657ccdeb0919a803c00c8cbd.md [hunk 2]
```diff
@@ -11,9 +11,9 @@ Finally, this is Linda, our UX/UI designer. She joined the team last year.
 
 # --description--
 
-`Last year` refers to the year before the current one. For example, if it is 2023 now, last year was 2022. `This year` refers to the current year, 2023 in this example. 
+`Last year` refers to the year before the current one. For example, if it is 2024 now, last year was 2023. `This year` refers to the current year, 2024 in this example. 
 
-`Next year` will be the year after the current one, which would be 2024 in this case. Understanding these expressions helps in placing events correctly in time.
+`Next year` will be the year after the current one, which would be 2025 in this case. Understanding these expressions helps in placing events correctly in time.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657ccdeb0919a803c00c8cbd.md:16]:
```suggestion
`Next year` will be the year after the current one, which would be 2034 in this case. Understanding these expressions helps in placing events correctly in time.
```

**Review by gikf — APPROVED** [review]

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657ccdeb0919a803c00c8cbd.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ Finally, this is Linda, our UX/UI designer. She joined the team last year.
 
 # --description--
 
-`Last year` refers to the year before the current one. For example, if it is 2024 now, last year was 2023. `This year` refers to the current year, 2024 in this example. 
+`Last year` refers to the year before the current one. For example, if it is 2033 now, last year was 2032. `This year` refers to the current year, 2033 in this example.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657ccdeb0919a803c00c8cbd.md:14]:
```suggestion
`Last year` refers to the year before the current one. For example, if it is 2043 now, last year was 2042. `This year` refers to the current year, 2043 in this example. 
```

sprry, brain fart on my side, we said 20 years not 10

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657ccdeb0919a803c00c8cbd.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ Finally, this is Linda, our UX/UI designer. She joined the team last year.
 
 # --description--
 
-`Last year` refers to the year before the current one. For example, if it is 2024 now, last year was 2023. `This year` refers to the current year, 2024 in this example. 
+`Last year` refers to the year before the current one. For example, if it is 2033 now, last year was 2032. `This year` refers to the current year, 2033 in this example. 
 
 `Next year` will be the year after the current one, which would be 2025 in this case. Understanding these expressions helps in placing events correctly in time.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657ccdeb0919a803c00c8cbd.md]:
```suggestion
`Next year` will be the year after the current one, which would be 2045 in this case. Understanding these expressions helps in placing events correctly in time.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657ccdeb0919a803c00c8cbd.md [hunk 5]
```diff
@@ -11,9 +11,9 @@ Finally, this is Linda, our UX/UI designer. She joined the team last year.
 
 # --description--
 
-`Last year` refers to the year before the current one. For example, if it is 2023 now, last year was 2022. `This year` refers to the current year, 2023 in this example. 
+`Last year` refers to the year before the current one. For example, if it is 2033 now, last year was 2032. `This year` refers to the current year, 2033 in this example. 
 
-`Next year` will be the year after the current one, which would be 2024 in this case. Understanding these expressions helps in placing events correctly in time.
+`Next year` will be the year after the current one, which would be 2034 in this case. Understanding these expressions helps in placing events correctly in time.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657ccdeb0919a803c00c8cbd.md:16]:
```suggestion
`Next year` will be the year after the current one, which would be 2044 in this case. Understanding these expressions helps in placing events correctly in time.
```

**Comment by ilenia-magoni** [comment]
You don't need to keep merging unless you have merge conflicts

**Review by majestic-owl448 — APPROVED** [review]

**Comment by ilenia-magoni** [comment]
Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 🎉

**Pull request merged**